crates/node/src/service.rs

```diff
@@ -527,6 +531,15 @@ pub fn new_full(
         );
     }
 
+    if let Some(ref l1_messages_config) = l1_messages {
+        let config = L1MessagesWorkerConfig::new_from_file(l1_messages_config)
+            .map_err(|e| sc_service::error::Error::Other(e.to_string()))?;
+        task_manager.spawn_essential_handle().spawn(
+            "l1-messaging",
```

Comment:
"l1-messaging` can be understood in both way l1 to l2 or l2 to l1. We want to be more explicit "ethereum-core-contract-events-listener"

Reply:
or something along those lines